test_fixtures/test_project_cnode_ref_generated/example.c

```diff
@@ -0,0 +1,101 @@
+#include "example.h"
```

Comment:
Something is off with catalogue structure. Shouldn't the fixtures be in test/fixtures.
Also, I think the projects are doubled: test_project_cnode occurs twice in repo, on top level and under test_fixtures

Reply:
I don't see the test_project_cnode under test_fixtures... https://github.com/membraneframework/unifex/tree/cnode_support/test_fixtures

Reply:
It's called `test_project_cnode_ref_generated` under fixtures. Now I see the difference, but still don't get why there are 3 additional catalogues at top level (outside the `test` directory)

Reply:
if you place another project within the test directory, the tests from that project are run when you run the tests from the original project, and they simply fail

Reply:
You can avoid that by either excluding those tests or changing `:test_paths` - https://hexdocs.pm/mix/Mix.Tasks.Test.html#module-configuration.
Aside from that, even at top level, the projects should be grouped under one dir (e.g. `test_project` -> `test_projects/nif`, `test_project_cnode -> test_proects/cnode`)
Fixtures do not contain tests so you can safely move  `test_fixtures/*` -> `test/fixtures/*`

Reply:
done